week1/homework/src/server.js

```diff
@@ -5,11 +5,40 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
 function createServer(port) {
   let state = 10;
 
+  // Created a function in order to not repeating myself.
+  function showResponse(response) {
+    response.setHeader('Content-Type', 'application/json');
+    response.write(JSON.stringify({ state }));
```

Comment:
as a tip: you are using a variable that exists somewhere else in the code.
this works fine, but you want your functions to be as independent as possible to reduce bugs and make them easier to understand.
I would pass `state` to the function 👍